### PR TITLE
[NUI] Add DisposeTest, Add DebugFileLogging Test

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Add DisposeTest, Add DebugFileLogging Test
- to solve https://code.sec.samsung.net/jira/browse/GRE-2372
- fix the bug that TextRunner was not working properly.
- change the TextRunner to executed one by one by using EventThread trigger.
- previously, it was reculsively executed in one timer callback.
- this means it accumulated call stack very much.

### API Changes ###
none

### CODE REVIEW

File: test/Tizen.NUI.Devel.Tests.Ubuntu/Tizen.NUI.Devel.Tests/Program.cs [hunk 1]
```diff
@@ -82,25 +88,47 @@ protected override void OnCreate()
...
+            trunner.LoadTestsuite();
 
-            timer = new Timer(1000);
-            timer.Tick += OnTick;
-            timer.Start();
+            asyncThreadMgr = TAsyncThreadMgr.GetInstance();
+            methodExecutionResetEvent = asyncThreadMgr.GetMethodExecutionResetEvent();
+            methodExecutionResetEvent.Reset();
 
+            Task t = Task.Run(() =>
```
Comment (everLEEst) [inline test/Tizen.NUI.Devel.Tests.Ubuntu/Tizen.NUI.Devel.Tests/Program.cs:99]: could be better to be named what task for?

**Review by everLEEst — APPROVED** [review]
Nice test. thank you :)

**Review by jaehyun0cho — APPROVED** [review]
LGTM

File: test/Tizen.NUI.Devel.Tests.Ubuntu/Tizen.NUI.Devel.Tests/Program.cs (hunk 1 above)
Comment (dongsug-song) [reply test/Tizen.NUI.Devel.Tests.Ubuntu/Tizen.NUI.Devel.Tests/Program.cs:99]: Thank you for review, I will change it in next patch.

**Pull request merged**